opencart2.3.x/upload/catalog/controller/extension/payment/xendit.php

```diff
@@ -156,6 +163,7 @@ private function process_order($response, $order_id) {
             $this->response->setOutput($message);
         } else {
             $message = 'Invoice not paid or settled. Cancelling order. Charge id: ' . $response['id'];
```

Comment:
Please change text to: 'Invoice not paid or settled. Cancelling order. **Invoice ID**: '